docs/contributions.rst

```diff
@@ -221,6 +221,16 @@ I like GNU style, but for keeping consistency in existing code of
 Exuberant-ctags, the origin of Universal-ctags, I introduced the style
 and configuration to my .emacs.  Please, do the same.
 
+Compatibility
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. TODO: to be removed?
```

Comment:
This is useful information.

Reply:
I can't remember why I thought this section was verbose.  I delete the TODO comment.